numba_cuda/numba/cuda/cudadrv/driver.py

```diff
@@ -2901,7 +2922,143 @@ def complete(self):
         """
 
 
-class MVCLinker(Linker):
+class _Linker(_LinkerBase):
+    def __init__(
+        self,
+        max_registers=None,
+        lineinfo=False,
+        cc=None,
+        lto=None,
+        additional_flags=None,
```

Comment:
All the arguments are only used as positional arguments - the defaults here imply they could be omitted, which isn't the case (for example the `arch = ` line below will fail if you omit `cc`) - perhaps these should all be positional?

Reply:
I think `cc` is the only parameter that there's no sensible default value for. Shall I make it required?

Reply:
Actually, I think we should just leave this as-is.